virt/virt-4-10-release-notes.adoc

```diff
@@ -143,7 +143,7 @@ Some features in this release are currently in Technology Preview. These experim
 * You can xref:../virt/node_network/virt-updating-node-network-config.adoc#capturing-nic-static-ip_virt-updating-node-network-config[transfer the static IP configuration of the NIC attached to the bridge] by applying a single `NodeNetworkConfigurationPolicy` manifest to the cluster.
 
 //CNV-16639
-* You can now install {VirtProductName} on IBM Cloud bare-metal servers. Bare-metal servers offered by other cloud providers are not supported.
+* You can now link:https://access.redhat.com/articles/6738731[install {VirtProductName} on IBM Cloud bare-metal servers]. Bare-metal servers offered by other cloud providers are not supported.
```

Comment:
Ah, I didn't realize that "IBM Cloud Bare Metal Servers" was the official name of the offering. I changed the capitalization in addition to nixing the hyphen. Great catch!

Reply:
If you look lower down on the IBM site, they say "IBM Cloud bare metal servers". So it seems like they use "bare metal servers" as generic terms, without hyphenation. I checked that because I was wondering whether it was "IBM Cloud Bare Metal Servers" (product name) or "IBM Cloud bare metal servers" (product name + generic terms). It seems to be the latter. In all cases, they did not use hyphens. Since that contradicts general English usage and ISG, I assume it's a corporate decision. 

Re: IBM terminology DB. Ingrid Towey responded yesterday on the CCS gchat: "we don't have access to the terminology database yet. The IBM Style Council has been working on getting us access to all the links in the guide,  but we don't have that yet." FYI.

Reply:
@ousleyp You're correct! It is capitalized. I misread the text. Thanks.